### ISSUE TITLE
Increase Default Color Contrast for Icons

### ISSUE THREAD

**Issue by rlbradey** [grommet/grommet#465]
The default color contrast  for icons is 2.85 and ideally should be 4.5.

**Comment by L0ZZI** [comment]
Thanks for the heads up @rlbradey! This is a defect we're working on soon. Grommet is aiming at WCAG AA compliance which is a 3.0:1 contrast ratio. We are "hopefully" hitting this ratio for other styled elements like text and buttons. Let me know if we missed anything else. ;-)

**Comment by ericsoderberghp** [comment]
Fixed in commit ef7fbb3c3209a21a1e4d562d5f427980a8c948df.
